data/common.yaml

```diff
@@ -61,8 +61,18 @@ scriptworker_users:
       worker_type_prefix: "tb-"
       worker_id_suffix: "tb"
       cot_product: "thunderbird"
+      # TODO: confirm if we need widevine for tb
       widevine_filename: "widevine_dep.crt"
       keychain_filename: "dep-signing.keychain"
+    vpnbld:
+      home: /builds/vpn-dep
+      supported_behaviors:
+        - mac_sign_and_pkg_vpn
+      dmg_prefix: "dep1"
```

Comment:
I believe we have these `dmg_prefix`es to avoid multiple scriptworkers on the same host from trying to clobber each other. https://github.com/mozilla-releng/scriptworker-scripts/blob/3fc292ffca72364787d2ddd7e7c6e1220b2d7f54/iscript/src/iscript/mac.py#L520

(Maybe we should use a tempdir?)

That would suggest we want the dmg_prefix to be `vpn` or `vpn1` instead of conflicting with depbld1's `dmg_prefix`.